{{ cookiecutter.format }}/briefcase.toml

```diff
@@ -3,12 +3,14 @@
 app_path = "src/app"
 app_packages_path = "src/app_packages"
 support_path = "src"
+{# Minor versions for 3.8, 3.9, and 3.10 cannot be bumped further -#}
+{# since Python is not hosting embeddable zipped versions of them -#}
```

Comment:
FYI - that's part of the normal service life of a Python release - they're in "security maintenance only" mode, so binaries are not produced. This is part of the process of encouraging people to upgrade.

Reply:
right; this note is mostly to help remind myself these can't be bumped anymore....because this is at least the second time Ive tried to before CI fails lol :)